app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -38,16 +38,16 @@
     private String sessionTitle;
 
     @ColorRes
-    private int sessionVividColorResId;
+    private int sessionVividColorResId = TopicColor.NONE.vividColorResId;
```

Comment:
Sorry...
It was when the language was not set.

So, if the language is null, I need to set the default value (`R.string.lang_en`)